.claude/hooks/pre_tool_use.py

```
@@ -0,0 +1,142 @@
+#!/usr/bin/env -S uv run --script
+# /// script
+# requires-python = ">=3.8"
+# ///
+
+import json
+import sys
+import re
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)